.github/workflows/govulcheck.yaml

```
@@ -0,0 +1,22 @@
+name: Run govulncheck
+on:
+   schedule:
+    - cron: "0 12 * * *" # 8am EST / 10am PST / 12pm UTC
+   workflow_dispatch: # allow manual trigger for testing
+
+jobs:
+  govulncheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
+        with:
+          go-version-file: go.mod
+          cache: false
+
+      - name: Install govulncheck
+        run: go install golang.org/x/vuln/cmd/govulncheck@latest
+
+      - name: Scan source code for known vulnerabilities
+        run: govulncheck -test ./...
```

I'll add the bot token in the repo.
```suggestion
        run: govulncheck -test ./...
        
      - name: Post to slack
        if: failure() && github.event_name == 'schedule'
        uses: slackapi/slack-github-action@91efab103c0de0a537f72a35f6b8cda0ee76bf0a # v2.1.1
        with:
          method: chat.postMessage
          token: ${{ secrets.GOVULNCHECK_BOT_TOKEN }}
          payload: |
            {
              "channel": "C08FGKZCQTW",
              "blocks": [
                {
                  "type": "section",
                  "text": {
                    "type": "mrkdwn",
                    "text": "Govulncheck failed in ${{ github.repository }}"
                  },
                  "accessory": {
                    "type": "button",
                    "text": {
                      "type": "plain_text",
                      "text": "View results"
                    },
                    "url": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
                  }
                }
              ]
            }
```